### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by wangxy518**
update an url

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

File: contributors/guide/contributing.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Note: if your pull request isn't getting enough attention, you can use the [#pr-
 - Make sure you include a clear and detailed PR description explaining the reasons for the changes, and ensuring there is sufficient information for the reviewer to understand your PR.
 - Additional Readings: 
     - [chris.beams.io/posts/git-commit/](https://chris.beams.io/posts/git-commit/)
-    - [github.com/blog/1506-closing-issues-via-pull-requests ](https://github.com/blog/1506-closing-issues-via-pull-requests)
+    - [github.com/blog/1506-closing-issues-via-pull-requests ](https://github.blog/2013-05-14-closing-issues-via-pull-requests/)
```
Comment (mrbobbytables) [inline contributors/guide/contributing.md:99]:
```suggestion
    - [GitHub Blog: Closing Issues via Pull Request](https://github.blog/2013-05-14-closing-issues-via-pull-requests/)
```
The link text itself contains the link, I'd update it either be generic (as in the suggestion above) or switch it to be the link and match the other links.

**Pull request closed without merging**